tasks/module_4/task_1.py

```diff
@@ -0,0 +1,65 @@
+"""Module4/task1: MadLib class consisting of text replacement utils."""
+import os
+
+import pyinputplus as pyip
+
+
+PLACEHOLDERS = ['ADJECTIVE', 'NOUN', 'ADVERB', 'VERB']
+
+
+class MadLib:
+    """Class used for replacing strings in files."""
+
+    def __init__(self, filename):
+        """:param filename: path to the input file."""
+        self.input = self.load_input_text(filename)
+
+        self.replacements = {}
+        self.output = ''
+
+    @staticmethod
+    def load_input_text(filename) -> str:
+        """Load text from input file.
+
+        :param filename: path to the input file.
+        :return text from file.
+        """
+        return open(filename, 'r').read()
+
+    def replace_placeholders(self):
+        """Replace placeholders and set output instance variable."""
+        output = self.input
+        for check, replacer in list(self.replacements.items()):
```

Comment:
It is not necessary to convert to list here. The `dict_items` object returned by `.items()` can be iterated just fine.